operators/flux/0.36.0/manifests/helmrelease.crd.yaml

```diff
@@ -0,0 +1,887 @@
+                                        Version and Kind it is capable of
+                                        unambiguously identifying and/or
+                                        selecting resources.
+                                        https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/api-group.md
+                                      type: string
+                                    kind:
+                                      description: Kind of the API Group to select resources from. Together with Group
+                                        and Version it is capable of
+                                        unambiguously identifying and/or
+                                        selecting resources.
+                                        https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/api-group.md
+                                      type: string
+                                    labelSelector:
+                                      description: LabelSelector is a string that follows the label selection
+                                        expression
+                                        https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#api
+                                        It matches with the resource labels.
+                                      type: string
+                                    name:
+                                      description: Name to match resources with.
+                                      type: string
+                                    namespace:
+                                      description: Namespace to select resources from.
+                                      type: string
+                                    version:
+                                      description: Version of the API Group to select resources from. Together with
+                                        Group and Kind it is capable of
+                                        unambiguously identifying and/or
+                                        selecting resources.
+                                        https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/api-group.md
+                                      type: string
+                                  type: object
+                              required:
+                                - patch
+                                - target
+                              type: object
+                            type: array
+                          patchesStrategicMerge:
+                            description: Strategic merge patches, defined as inline YAML objects.
+                            items:
+                              x-kubernetes-preserve-unknown-fields: true
+                            type: array
+                    type: object
```

Comment:
I'm not sure how these CRD's were generated, but they are not identical to the community ones. Look at the nesting in the community one on the right

![image](https://user-images.githubusercontent.com/88156657/199605931-36b96fc9-e33e-42db-be15-be524a7e4486.png)

Reply:
```
E1102 21:15:12.557300       1 queueinformer_operator.go:290] sync {"update" "flux-system/install-qrx7m"} failed: CustomResourceDefinition.apiextensions.k8s.io "helmreleases.helm.toolkit.fluxcd.io" is invalid: spec.validation.openAPIV3Schema.properties[spec].properties[postRenderers].items.properties[kustomize].type: Required value: must not be empty for specified object fields
```